src/resolvers.ts

```diff
@@ -53,6 +53,7 @@ const resolveContentPaths = (srcDir: string, nuxtOptions = useNuxt().options) =>
     r(`{A,a}pp${sfcExtensions}`),
     r(`{E,e}rror${sfcExtensions}`),
     r(`app.config${defaultExtensions}`),
+    r(`spa-loading-template.html`),
```

Comment:
how about adding this only when `nuxtOptions.ssr === false`?

Reply:
and `spaLoadingTemplate !== false`

Reply:
but does the HTML include `nuxt.options.css` in head?

Reply:
> how about adding this only when `nuxtOptions.ssr === false`?

Wouldn't `nuxtOptions.ssr === false` mean it is only included when ssr is turned off for the entire application?

Reply:
> and `spaLoadingTemplate !== false`

I guess a check for `nuxtOptions.spaLoadingTemplate !== false` should work, yes. I thought since `error.vue` e.g. is included without checking if the file exists in the project that a simple list of all possible files is preferred, otherwise I can add this check :)

Reply:
So `nuxtOptions.spaLoadingTemplate` can be a `string` as well i.e., a custom name for `spa-loading-template.html` possibly - but it will be applied when `ssr: false`

Reply:
> but does the HTML include `nuxt.options.css` in head?

Yes, the generated html includes the tailwind css bundle

Reply:
ah, I didn't realize a custom file name could be passed... I will check this and the ssr option asap and report back

Reply:
Okay, I'm a little bit confused. When I turn off ssr for the entire application, the styles do not get included, but when I use routeRules the styles get injected. I've only tested this in the playground for now.

Reply:
Just pushed - seems to be working OK; will need to check the appDir!